### PR TITLE
Updating VSTest nuget

### PR DESCRIPTION
**Pull request by eddynaka**
Fixes #.

## Changes

Please provide a brief description of the changes here. Update the
`CHANGELOG.md` for non-trivial changes.

For significant contributions please make sure you have completed the following items:

* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: build/Common.props [hunk 1]
```diff
@@ -29,7 +29,7 @@
     <MicrosoftAspNetCoreHttpFeaturesPkgVer>[2.1.1,5.0)</MicrosoftAspNetCoreHttpFeaturesPkgVer>
     <MicrosoftAspNetTelemetryCorrelationPkgVer>[1.0.7,2.0)</MicrosoftAspNetTelemetryCorrelationPkgVer>
     <MicrosoftCodeAnalysisFxCopAnalyzersPkgVer>[3.3.0-beta2.final]</MicrosoftCodeAnalysisFxCopAnalyzersPkgVer>
-    <MicrosoftCodeCoveragePkgVer>[16.6.1]</MicrosoftCodeCoveragePkgVer>
+    <MicrosoftCodeCoveragePkgVer>[16.8.0-preview-20200812-03]</MicrosoftCodeCoveragePkgVer>
```
Comment (cijothomas) [inline build/Common.props:32]: any reason why we want to move to preview version?

Comment (eddynaka) [reply build/Common.props:32]: Some time ago (a week maybe), I tried the version `16.7.0`. But that version came with a bug, which I reported.
Now, they launched the preview and asked me to test again. If that works fine, they will release a stable one shortly.

So, in the end we will use the version `16.7.1`.

**Comment by eddynaka** [comment]
> LGTM

dont merge please...we are waiting the new version to be released.

**Comment by CodeBlanch** [comment]
@eddynaka 10-4

**Pull request merged**